### PR TITLE
Set ETH_CALL_DEFAULT_TO_CONSENSUS_NODE to false

### LINKED ISSUE #1311: Release 0.25.0
### Problem

0.25.0 features are not yet deployed

### Solution

manual release process
- [x] Create a release/0.25 branch off of main. Ensure github test actions run
- [x] Tag as ```v0.25.0-rc1```
     - [x] git tag
     - [x] Confirm new docker image version is deployed
- [x] Previewnet Testing
     - [x] Deploy tagged version
     - [x] Manual testing
     - [x] Run newman tests
     - [x] Run Dapp Example Bootstrap and manual tests
     - [x] Run acceptance tests
     - [x] Run performance tests
- [x] Testnet Testing
     - [x] Deploy tagged version
     - [x] Manual testing
     - [x] Run newman tests
     - [x] Run Dapp Example Bootstrap and manual tests
     - [x] Run acceptance tests
     - [ ] Run performance tests
 - [x] Let Bake
 - [x] Tag as ```v0.25.0```
     - [x] git tag
     - [x] Confirm new docker image version is deployed
     - [x] Write up release notes and changelist
 - [x] Mainnet Testing
     - [x] Deploy tagged version
     - [x] Manual testing
 
 Any bugs or missed features found should see a new ticket opened, addressed in main and cherry-picked to ```release/0.25```  with a new rc version tagged and docker image deployed   

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by ebadiere**
**Description**:
Set ETH_CALL_DEFAULT_TO_CONSENSUS_NODE to false.

Fixes #

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG
Needs `eth_call` logic update

File: charts/hedera-json-rpc-relay/values.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ config:
   DEV_MODE: false
   DEFAULT_RATE_LIMIT: 200
   ETH_CALL_CACHE_TTL: 200
-  ETH_CALL_DEFAULT_TO_CONSENSUS_NODE: true
+  ETH_CALL_DEFAULT_TO_CONSENSUS_NODE: false
```
[the remark this answers is not in the document]
Comment (ebadiere) [reply charts/hedera-json-rpc-relay/values.yaml:21]: Updated.

File: packages/relay/tests/lib/openrpc.spec.ts [hunk 2]
```diff
@@ -202,6 +202,21 @@ describe("Open RPC Specification", function () {
         process.env.ETH_CALL_DEFAULT_TO_CONSENSUS_NODE = initialEthCallConesneusFF;
     });
 
+    it('should execute "eth_call" against mirror node with an undefined ETH_CALL_DEFAULT_TO_CONSENSUS_NODE', async function () {
```
[the remark this answers is not in the document]
Comment (ebadiere) [reply packages/relay/tests/lib/openrpc.spec.ts:205]: There was already a set of two tests with all the necessary set up called 'should execute "eth_call" against mirror node' and 'should execute "eth_call" against consensus node' that explicitly set the process.env.ETH_CALL_DEFAULT_TO_CONSENSUS_NODE in the openrpc.spec.ts.

[the remark this answers is not in the document]
Comment (ebadiere) [reply packages/relay/tests/lib/openrpc.spec.ts]: Moved all three tests to eth.spec.ts and added a sinon check.

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**